apps/graphql/src/queries/__tests__/__snapshots__/queries.test.js.snap

```diff
@@ -92,74 +92,74 @@ fragment RouteStopFragment on RouteStop {
       "edges": [
         {
           "node": {
-            "id": "Q3VzdG9tZXJCb29raW5nOjI3MDcyNTE=",
-            "status": "confirmed",
+            "id": "Qm9va2luZ09uZVdheToxNjQ2MzQ0Nw==",
```

Comment:
Are these mocks auto-updated? Where do they come from?

Reply:
I just added all bookings from the `reactnative@kiwi.com` account, since we have better data there. More bookings, and different types. The old set just contained 3 bookings, all OneWay